modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -1983,6 +1983,43 @@ describe('V2 Wallet:', function () {
         args[1]!.should.equal('full');
       });
 
+      it('should call prebuildTxWithIntent with the correct params for eth accelerations for receive address', async function () {
+        const recipients = [{
+          address: '0xAB100912e133AA06cEB921459aaDdBd62381F5A3',
+          amount: '1000',
+          tokenName: 'gterc18dp',
+        }];
+
+        const feeOptions = {
+          maxFeePerGas: 3000000000,
+          maxPriorityFeePerGas: 2000000000,
+        };
+
+        const lowFeeTxid = '0x6ea07f9420f4676be6478ab1660eb92444a7c663e0e24bece929f715e882e0cf';
+        const receiveAddress = '0x062176bc9345da3e8ee90361b0cf6ff883ba7206';
+
+        const prebuildTxWithIntent = sandbox.stub(ECDSAUtils.EcdsaUtils.prototype, 'prebuildTxWithIntent');
+        prebuildTxWithIntent.resolves(txRequestFull);
+
+        await tssEthWallet.prebuildTransaction({
+          reqId,
+          recipients,
+          type: 'acceleration',
+          feeOptions,
+          lowFeeTxid,
+          receiveAddress,
+        });
+
+        sinon.assert.calledOnce(prebuildTxWithIntent);
+        const args = prebuildTxWithIntent.args[0];
+        args[0]!.should.not.have.property('recipients');
+        args[0]!.feeOptions!.should.deepEqual(feeOptions);
+        args[0]!.lowFeeTxid!.should.equal(lowFeeTxid);
+        args[0]!.receiveAddress!.should.equal(receiveAddress);
+        args[0]!.intentType.should.equal('acceleration');
+        args[1]!.should.equal('full');
```

Comment:
can we have any assertions on the recipients array params being present in the prebuilt response? or the token name/id even?

Reply:
As mentioned above, here we actually checking that recipients should not be there in the response. Apart from this as discussed I have also added a check where for intentType fillNonce and acceleration, users will be thrown an error if they pass recipients